index.md

```diff
@@ -82,6 +82,8 @@ _For those thinking: “We don’t have anything yet, where do we start?”_
 * [Inclusion Ventures](http://inclusionventures.com/), a strategic consulting firm that provides training to help organizations and VCs take an inclusive approach to diversity and leadership
 * [FemaleFounder.org](https://www.femalefounder.org/), a community of women helping women (e.g., female founder office hours hosted across the country with top-tier female VCs)
 * [Parity.org](https://www.parity.org/), an organization that encourages corporations to commit to interviewing at least one qualified woman for all open roles at the VP level and above
+* [ReadySet](http://www.thereadyset.co/), is a strategy and solutions firm that helps innovative organizations become more diverse, equitable, and inclusive 
+* [OutsideAssist](https://outsideassist.com/) deploys both the reporting tools and the mediation team needed to help businesses identify and solve human issues early and fairly.
```

Comment:
Same comments as for ReadySet, make this 1. not a sentence 2. less marketing. Maybe something like: "OutsideAssist, a software and services firm that provides both reporting tools and a mediation team for identifying and solving HR issues"